source/localizable/routing/defining-your-routes.md

```diff
@@ -84,7 +84,18 @@ the `{{outlet}}` of the `posts` template.
 A nested route's names includes the names of its ancestors.
 If you want to transition to a route (either
 via `transitionTo` or `{{#link-to}}`), make sure to use the full route
-name (`posts.new`, not `new`).
+name (`posts.new`, not `new`). If you don't want to use the full route
+name (e.g., because a resource can be accessed in multiple locations), use the
```

Comment:
Should I change it to something like `If you don't want to use that full name, you can refer to it just by the name of the child with the resetNamespace option (but note that this does not change the URL structure).`?

Or if you really think that `resetNamespace` should be used rarely, then it should be removed from the final example on emberjs.com.